SPECS/xcp-ng-xapi-storage.spec

```diff
@@ -1,23 +1,54 @@
 Name:           xcp-ng-xapi-storage
 Version:        1.1.0
-Release:        1%{?dist}
+Release:        1.0.zfsvol.0%{?dist}
```

Comment:
```suggestion
Release:        1.0.zfsvol.1%{?dist}
```

Reply:
any problem with `0`?

Reply:
As long as we don't build it. The first test build in a repo managed by koji would be `1.0.zfsvol.1%{?dist}`.